package.json

```diff
@@ -10,7 +10,7 @@
     "flow:stop": "flow stop",
     "flow": "flow; test $? -eq 0 -o $? -eq 2",
     "lint": "eslint src/",
-    "prepublish": "npm run lint && npm run flow && npm run test && npm run build",
```

Comment:
I think you didn't wanted to commit this.

Reply:
Agreed.